jobsapp/api/views/common.py

```diff
@@ -27,3 +30,23 @@ def get_queryset(self) -> "QuerySet[Job]":
             )
         else:
             return self.serializer_class.Meta.model.objects.filter(filled=False)
+
+
+@api_view(["GET"])
+@permission_classes(
+    [
+        AllowAny,
+    ]
+)
+def about_us(request: Request) -> Response:
```

Comment:
@francescarpi which information should we share here?
@XaviTorello may you help me here providing the correct information for each field?

My approach here will be create a new model, for example `AboutUs`, and add it to the admin page, so we can change the information from there. It's okay or there is a better approach?